cloudsim-plus/src/main/java/org/cloudbus/cloudsim/hosts/HostSimple.java

```diff
@@ -329,6 +327,18 @@ public static void setDefaultStorageCapacity(final long defaultCapacity) {
         defaultStorageCapacity = defaultCapacity;
     }
 
+    @Override
+    public void setStartUpShutDownDelay(final double hostStartUpDelay, final double hostShutDownDelay) {
+        if(hostStartUpDelay > 0) this.powerModel.setHostStartupDelay(hostStartUpDelay);
+        if(hostShutDownDelay > 0) this.powerModel.setHostShutDownDelay(hostShutDownDelay);
+    }
+
+    @Override
+    public void setStartUpShutDownPower(final double hostStartUpPower, final double hostShutDownPower) {
+        if(hostStartUpPower > 0) this.powerModel.setHostStartupPower(hostStartUpPower);
```

Comment:
There is no need for those `if` conditions here. If the given value is zero and the current attribute is also zero, that doesn't change anything. And there is no worries about performance here. This way, as less code we have as better. 

Furthermore, the `if` may cause unexpected behavior. If I set the attributes with some value and later on I dedice to disable them (setting 0), the condition won't allow me to change them to zero again.